spec/requests/v0/in_progress_forms_controller_spec.rb

```diff
@@ -131,6 +131,10 @@
     end
 
     describe '#show' do
+      before do
+        Flipper.disable('remove_pciu_2')
+      end
+
       let(:user) { build(:user, :loa3, address: build(:mpi_profile_address)) }
```

Comment:
Flipper will be tested in the form_profile_v2_spec in this https://github.com/department-of-veterans-affairs/vets-api/pull/18729

Reply:
aren't we trying to move towards mocking Flipper instead of using before actions, because it sets it globally and introduces inconsistencies?

Reply:
This is why we need to fix these specs. I attempted to mock the flipper but due to contact information being used throughout the app and all the flipper inconsistencies, mocking will only give more errors. I even attempted to try to re-work other specs but it the work kept going.

Reply:
I accidentally made a [duplicate ticket](https://github.com/orgs/department-of-veterans-affairs/projects/1335/views/1?pane=issue&itemId=91668643) over this issue when I was drafting this PR.